proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -231,8 +238,11 @@ service BeaconChain {
         };
     }
 
+    // DEPRECATED: Streaming Validator Info is nolonger supported, the /eth/v1/events Beacon API endpoint will stream events and
+    // the /eth/v1/beacon/states/{state_id}/validators Beacon API endpoint will provide validator info.
```

Comment:
I wouldn't mention the events endpoint here, because it does not help with validators.